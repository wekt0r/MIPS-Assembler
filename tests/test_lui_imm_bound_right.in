lui $s0, -10
lui $s0, 30000
lui $s1, 45000
