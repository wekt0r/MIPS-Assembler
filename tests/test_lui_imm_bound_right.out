.text
00000000	3C10FFF6	lui	$s0,-10
00000004	3C107530	lui	$s0,30000
00000008	3C11AFC8	lui	$s1,45000
